core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -545,6 +546,12 @@ class RDDSuite extends FunSuite with SharedSparkContext {
     assert(sortedTopK === nums.sorted(ord).take(5))
   }
 
+  test("isEmpty") {
+    assert(sc.emptyRDD.isEmpty())
+    assert(sc.parallelize(Seq[Int]()).isEmpty())
+    assert(!sc.parallelize(Seq(1)).isEmpty())
```

Comment:
I think the `sc.parallelize(Seq[Int]()` case actually has multiple partitions but I'll add this too. Also, I'll check the case where the first partition is empty but others aren't.